src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -144,10 +147,6 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         return { ...item, permissions };
     };
 
-    getMetadataColumnName(column: MetadataColumnConfig | string): string {
-        return typeof column === 'string' ? column : getProp(column, 'name');
-    }
-
```

Comment:
Column/Field names will be determined by `displayName` of the result, and not by any prop.